extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/Lib485BridgeHandler.java

```diff
@@ -0,0 +1,143 @@
+    }
+
+    @Override
+    protected void openConnection() {
+        for (IpNode receiverNode : receiverNodes.keySet()) {
+            Socket socket = receiverNodes.get(receiverNode);
+            if (socket == null) {
+                try {
+                    socket = new Socket(receiverNode.getAddressString(), receiverNode.getPort());
+                } catch (IOException e) {
+                    logger.warn("Could not connect to {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                    return;
+                }
+            }
+
+            if (socket.isConnected()) {
+                receiverNodes.put(receiverNode, socket);
+            } else {
+                receiverNodes.put(receiverNode, null);
+                return;
+            }
+
+        }
+        isConnected = true;
+    }
+
+    @Override
+    protected void closeConnection() {
+        for (IpNode receiverNode : receiverNodes.keySet()) {
+            Socket socket = receiverNodes.get(receiverNode);
+            if ((socket != null) && (!socket.isClosed())) {
+                try {
+                    socket.close();
+                } catch (IOException e) {
+                    logger.warn("Could not close socket {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                }
+            }
+            receiverNodes.put(receiverNode, null);
+        }
+        isConnected = false;
+    }
+
+    @Override
+    protected void sendDmxData() {
+        long now = System.currentTimeMillis();
+        universe.calculateBuffer(now);
+        for (IpNode receiverNode : receiverNodes.keySet()) {
+            Socket socket = receiverNodes.get(receiverNode);
+            if (socket.isConnected()) {
+                try {
+                    socket.getOutputStream().write(universe.getBuffer());
+                } catch (IOException e) {
+                    logger.warn("Could not send to {} in {}: {}", receiverNode, this.thing.getUID(), e);
+                    isConnected = false;
+                    return;
+                }
+            } else {
+                isConnected = false;
+                return;
+            }
+        }
+    }
+
+    @Override
+    protected void updateConfiguration() {
+
+        Configuration configuration = getConfig();
+
+        universe = new Universe(MIN_UNIVERSE_ID);
+
+        receiverNodes.clear();
+        if (configuration.get(CONFIG_ADDRESS) == null) {
+            receiverNodes.put(new IpNode("localhost:9020"), null);
+            logger.debug("sending to {} for {}", receiverNodes, this.thing.getUID());
+        } else {
+            for (IpNode receiverNode : IpNode.fromString((String) configuration.get(CONFIG_ADDRESS), DEFAULT_PORT)) {
+                receiverNodes.put(receiverNode, null);
+                logger.debug("sending to {} for {}", receiverNode, this.thing.getUID());
+            }
+        }
+        super.updateConfiguration();
+
+        updateStatus(ThingStatus.ONLINE);
```

Comment:
yes, at least then you know that you can talk to the receivers. Though I am still a bit confused when you should set the bridge to "ONLINE". Since you have a UDP connection to the bridges you suggest to set them ONLINE immediately and "hope for the best". But then I think a `Thing` that is attached to the bridge can only be "ONLINE" if `isConnected` is `true` here.